### PR TITLE
Add URL encoding for filters on accounts/transfers pages

### PR DESCRIPTION
**Pull request by tristanhcole**
**Describe the Pull Request**

Screen Shot 2020-12-07 at 3 44 45 pm

Uses the same schema as #553. Merge in #553 first.

Updates `SearchBoxWithFilter.jsx` to use ant design. Changes delimiter from `:` to `|` to support dates on account page.
Deprecates saving/loading filters as will be updated to use URL query schema.

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes, checking [accessibility](https://www.a11yproject.com/checklist/) etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

**Comment by tristanhcole** [comment]
[Add filters to Transfer account page](https://trello.com/c/Ly0WAdqt/491-add-filters-to-transfer-account-page)

**Comment by tristanhcole** [comment]
[Add filters to Credit Transfer page](https://trello.com/c/OH4V6NUp/490-add-filters-to-credit-transfer-page)

File: app/client/components/organizationWrapper.jsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ const OrganizationWrapper = Component =>
         }
       } else {
         this.props.history.replace({
-          search: generateQueryString()
+          search: location.search
```
Comment (tristanhcole) [inline app/client/components/organizationWrapper.jsx:29]: This was causing url queries to be wiped if they weren't captured/read by a rendered component before the organisation wrapper wiped the query string.

File: app/client/utils.js [hunk 2]
```diff
@@ -280,7 +281,7 @@ export const get_zero_filled_values = (key, value_array, date_array) => {
 
 export const processFiltersForQuery = filters => {
   let encoded_filters = encodeURIComponent("");
-  let delimiter = ":";
+  let delimiter = "|";
```
Comment (tristanhcole) [inline app/client/utils.js:284]: created filter on accounts/transfer page uses `:` in encoded filter which causes URL error on load. This changes the delimter which doesn't have any impact across the app

Comment (enjeyw) [reply app/client/utils.js:284]: pipes aren't web safe

**Review by enjeyw — CHANGES_REQUESTED** [review]
Forgot to put my pipes comment as a review lol.

**Comment by tristanhcole** [comment]
No longer needed

**Pull request closed without merging**